src/solver/utils/optimization_statistics.h

```diff
@@ -28,6 +28,14 @@ class OptimizationStatistics
         this->reset();
     }
 
+    OptimizationStatistics(const OptimizationStatistics &o)
```

Comment:
See "[rule of 5](https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#c21-if-you-define-or-delete-any-copy-move-or-destructor-function-define-or-delete-them-all)": for clarity sake,
let's define also copy operator and move operators (or delete them).